ecs-cli/modules/cli/compose/factory/factory.go

```diff
@@ -58,9 +58,15 @@ func (projectFactory projectFactory) Create(cliContext *cli.Context, isService b
 
 // populateContext sets the required CLI arguments to the context
 func (projectFactory projectFactory) populateContext(ecsContext *context.Context, cliContext *cli.Context) error {
-	// populate CLI context
-	// TODO: Support multiple compose files
-	ecsContext.ComposeFiles = []string{cliContext.GlobalString(command.ComposeFileNameFlag)}
+	/*
+	  populate CLI context
+	  libcomposecommand.Populate updates the composeFiles field in the context
+	  with the files specified by --file (or -f), or if no files are specfied,
+	  it looks for a docker-compose.yml and a docker-compose.override.yml.
+	  We rely on libcompose for this functionality; it should correctly mimic
+	  the behavior of docker-compose.
+	*/
+	libcomposecommand.Populate(&ecsContext.Context, cliContext)
 	ecsContext.ProjectName = cliContext.GlobalString(command.ProjectNameFlag)
```

Comment:
libcomposecommand.Populate also seems to set the ProjectName. Is this needed here?